elpy.el

```diff
@@ -102,14 +103,27 @@ can be inidividually enabled or disabled."
   :group 'elpy)
 
 (defcustom elpy-project-ignored-directories
-  (append '(".tox" "build" "dist" ".cask" ".ipynb_checkpoints")
-          vc-directory-exclusion-list)
-  "Directories ignored by functions working on the whole project."
+  '(".tox" "build" "dist" ".cask" ".ipynb_checkpoints")
+  "Directories ignored by functions working on the whole project.
+This is in addition to `vc-directory-exclusion-list'
+and `grep-find-ignored-directories', as appropriate."
   :type '(repeat string)
   :safe (lambda (val)
           (cl-every #'stringp val))
   :group 'elpy)
 
+(defun elpy-project-ignored-directories ()
+  "Compute the list of ignored directories for project.
+Combines
+  `elpy-project-ignored-directories'
+  `vc-directory-exclusion-list'
+  `grep-find-ignored-directories'"
+  (cl-union
+   (append elpy-project-ignored-directories vc-directory-exclusion-list)
```

Comment:
Minor issue: the result of `elpy-project-ignored-directories` can have duplicate items due to this `append`.
would it be a problem to use `cl-union` in place to avoid this ?